### DISCUSSION TITLE
version 1.7.0 issue

### DISCUSSION THREAD

**Discussion by pouemes** [Leaflet/Leaflet#8321] [Q&A] [answered]
Hello
on cdn https://unpkg.com/leaflet@1.7.0/dist/leaflet.js seem not to be version 1.7.0 but 1.4.0
/* @preserve
 * Leaflet 1.4.0+crs-min-max-zoom.d7a7ba1, a JS library for interactive maps. http://leafletjs.com
 * (c) 2010-2018 Vladimir Agafonkin, (c) 2010-2011 CloudMade
 */

**Comment by Malvoz** [comment, marked as answer]
https://leafletjs.com/2020/09/04/leaflet-1.7.1.html:

> The release was supposed to be numbered as 1.7.0, but a problem with the NPM dependencies appeared at the last minute, generating a bad build. Version 1.7.0 has been marked as deprecated in NPM, and shouldn’t be used - use 1.7.1 instead.

**Comment by pouemes** [comment]
thanks i undestand now
i am also trying to solve this issue https://github.com/Leaflet/Leaflet/discussions/8322
scroll problem with firefox with leaflet version above 1.6.0
that why i was testing previous version
perhaps have you solution
